contributing/diversity/governance.rst

```diff
@@ -0,0 +1,110 @@
+Diversity Initiative Governance
+===============================
+
+Membership
+----------
+
+Membership of Symfony’s Diversity Initiative is open to any member of the
+Symfony community; to avoid the risk of elitism or meritocracy, no requirement
+is needed to be involved. All members, at any time, are invited to put forward
+ideas and suggestions as a proposal for an actionable item.
+
+Guidance
+--------
+
+The project leader, Fabien Potencier, is responsible for publicly appointing
+five (5) key players of the initiative to provide guidance and drive it forward,
+but also retains the right to revoke any of the key players at any time. These
+guiding key players should:
+
+* Be committed to the initiative’s cause and have joined because they want to
+  help the initiative to deliver its purpose most effectively for the
+  community’s benefit.
+* Recognize that meeting the initiative’s purpose is an ongoing effort.
+* Be committed to good governance and want to contribute to the initiative’s
+  continued improvement.
+
+Veto
+~~~~
+
+The project leader (Fabien Potencier) will have the right to veto any actionable
+item, regardless of the vote of the initiative’s key players. The project leader
+may, at their discretion, also appoint other people from among the initiative’s
+key players to also have the right to veto - in such a case these people are
+expected to use appropriate judgement to know when to use a “no” vote or a veto.
+Any single veto will reject an actionable item.
+
+The purpose of having members with the right to veto is to prevent a “people’s
+majority” from overruling the core interests of the Symfony project. This will
+encourage communication between proposing members, the initiative’s key players
+and the Core Team to create realistic proposals, and in return any veto will
+come with a full explanation (not just a justification).
+
+Advice Process
+~~~~~~~~~~~~~~
+
+When a proposal on an actionable item is ready to be decided on, insight from
+the community (advice, general consensus, or non-binding poll) should be
+requested from the wider community - this will aim to include both those who
+will be meaningfully affected and those with meaningful expertise in the matter
+at hand.
+This feedback will enable the key players to have the confidence to vote for the
+best possible decision according to the information they have available, knowing
+that the responsibility they accept for said vote is justified.
+
+Voting
+~~~~~~
+
+Key players have the right to vote on proposals for actionable items.
```

Comment:
We should clarify that players can only vote "yes" or "no". They cannot abstain or vote "null".

Reply:
I guess they could abstain, but given the quorum limits it would most likely be equivalent to a "no" vote. perhaps this should be reworded.

Reply:
basically if all 4 people vote and one person abstains and 2 people vote in favor and one against .. then the qorum would be passed and the result would be yes with over 50%

Reply:
OK, but please let's be explicit about the possible valid votes of people. Thanks.

Reply:
I don't think so. 2 out of 4 people is 50%, which is not **over** 50%. Or do you not count people that abstain? But in that case, should you count them towards the quorum?

Reply:
I think that the quorum should be *voting* people, and should not count abstains (same as not submitting a vote, not replying, or being unavailable). How about adding to the sentence below, something like:

> *"The quorum of "yes" or "no" votes required for a decision to be considered valid is at least 75% of active, appointment key players.

With 5 key players, and 3 options (yes, no, abstain), there are 125 different outcomes - I'll try to come up with an easy-to-digest way of presenting the outcomes of this (Reject or Pass).

Reply:
in my scenario I would count it as 80% quorum met and 66% votes in favor and 33% opposed. if it would be 50% then abstain would be the same as no.

but sure we can make this more explicit.

Reply:
I guess with 5 people, we could even list all possible permutations :)

Reply:
| Yes | No | Abstain | Result |
|-----|----|---------|--------|
| 5   | 0  | 0       | Pass   |
| 4   | 1  | 0       | Pass   |
| 4   | 0  | 1       | Pass   |
| 3   | 2  | 0       | Pass   |
| 3   | 1  | 1       | Pass   |
| 3   | 0  | 2       | Reject |
| 2   | 3  | 0       | Reject |
| 2   | 2  | 1       | Stalemate (Reject) |
| 2   | 1  | 2       | Reject |
| 2   | 0  | 3       | Reject |
| 1   | 4  | 0       | Reject |
| 1   | 3  | 1       | Reject |
| 1   | 2  | 2       | Reject |
| 1   | 1  | 3       | Reject |
| 1   | 0  | 4       | Reject |
| 0   | 5  | 0       | Reject |
| 0   | 4  | 1       | Reject |
| 0   | 3  | 2       | Reject |
| 0   | 2  | 3       | Reject |
| 0   | 1  | 4       | Reject |
| 0   | 0  | 5       | Reject |

Reply:
People that don't vote get an automatic "abstain"?

Reply:
Those results permutations listed above are for an abstain vote **not** counting towards the quorum.

It would add more complexity to count an abstained vote towards the quorum, as there'd be 4 options (yes, no, abstain, no reply and not counted towards quorum), though I'm not sure on the best course of action.

Reply:
To me, this feels a bit weird in this scenario:
3 votes for, 1 abstain and 1 yet to vote... If that last person picks "abstain", the vote will be rejected. However, if that person picks "no", the vote will pass.

Reply:
Unless players are only allowed to vote "yes" or "no". Then it would make sense, since you can't vote with 3 players or less.

Reply:
Yeah that is a bit weird. I think that an abstain vote is the same as not voting, ie. people can *only* vote **yes** or **no**.

Reply:
Okay, following this discussion I've updated the document for clarification.